README.md

```diff
@@ -8,17 +8,35 @@
 * [License](#License)
 
 ## Description
-I love music and frequently attend live music shows when I can, however I am sick of going to shows with uncertainty as to who I am seeing. I would like an app that allows me to search by band name, preview their music and see all upcoming events.
+Scenario 1
+* I love music and frequently attend live music shows when I can, however I am sick of going to shows with uncertainty as to who I am seeing. I would like an app that allows me to search by band name, preview their music and see all upcoming events.
+
+Scenario 2
+* As a music Promoter I am always looking to spread the word about upcoming gigs and shows. I need an application that allows users to sample various artists and easily figure out where and when these artists are performing.
 
 ## Usage
-* When loaded the page will:
-    * On the left hand side show a history of your last 5 searches and views.
-    * In the central panel introduce and explain the website.
-    * On the right hand side show a google map with your location.
-* Search for a band by key word.
-* View the list of results on the left and listen to 30 second song previews.
-* When an artist is clicked on the central panel will display artist information
-    * The map will show event location.
+* When loaded the page will display:
+    * Search and history on the left hand panel.
+    * Informatation about the website including future development opertunities in the central panel.
```

Comment:
```suggestion
    * Information about the website including future development oportunities in the central panel.
```

Reply:
Information about the website including future development opportunities in the central panel.

Reply:
Opportunities has a double 'p'